File: BedivereKnx.GUI/Forms/frmStartUp.vb

Public NotInheritable Class frmStartUp

    Private Sub frmStartUp_Load(sender As Object, e As EventArgs) Handles Me.Load
#If DEBUG Then
        'Me.Close()
#End If
        Me.Text = My.Application.Info.AssemblyName
        lblProdName.Text = My.Application.Info.ProductName
        lblVersion.Text = $"Ver {Application.ProductVersion}" '{My.Application.Info.Version}
        lblCopyright.Text = $"{My.Application.Info.Copyright}.{vbCrLf}All rights reserved."
    End Sub

End Class

File: BedivereKnx.GUI/Forms/FrmStartup.cs
namespace BedivereKnx.GUI.Forms
{
    public partial class FrmStartup : Form
    {

        public FrmStartup()
        {
            InitializeComponent();
            Text = Globals.AssemblyInfo.ProductName;
            lblProdName.Text = Globals.AssemblyInfo.ProductName;
            lblVersion.Text = $"Ver {Globals.AssemblyInfo.Version}";
            lblCopyright.Text = $"{Globals.AssemblyInfo.CopyRight}.{Environment.NewLine}All rights reserved.";
            lblAuth.Text = Globals.AuthInfo!.Title;
        }

    }

}
